webnode/src/components/overlay/index.tsx

```diff
@@ -1,13 +1,26 @@
-import React, { Component } from "react";
+import * as React from "react";
+import { bindActionCreators, Dispatch } from "redux";
 import { connect } from "react-redux";
 
 import nodeActions from "../../redux/actions/node-actions";
 import consentActions from "../../redux/actions/consent-actions";
 import ConsentOverlay from "../consent-overlay";
 
 import { CONSENT_STATUS } from "../../config";
+import { RootState } from "../../types";
+
+interface OverlayProps {
+  setOwnerEthAddress: (ethAddress: string) => any;
```

Comment:
This is a great example of what we get with types. Glad you didnt just throw a bunch of any's in there. 👍

Reply:
yeh i like this. should it return `void`?

Reply:
i was more commenting on the whole interface.